Files/ViewModels/MainViewModel.cs

```diff
@@ -146,11 +146,11 @@ private void DetectFontName()
 
         public static async void OpenSettings()
         {
-            if (!UIHelpers.IsAnyContentDialogOpen())
+            if (UIHelpers.IsAnyContentDialogOpen())
```

Comment:
Mmmm it looked correct before.

Reply:
It was the opposite of what I wanted 😂
We only want to open the settings dialog when there aren't any other dialogs open.

Reply:
It doesn't seem to have worked to fix the issue 🤔